### DISCUSSION TITLE
[高难警告] 这怎么打  这拿头打

### DISCUSSION THREAD

**Discussion by misaka10384** [share121/inter-knot#2132] [其他]
![902d6fb4150a34030c8b5cc6e904f1a3](https://github.com/user-attachments/assets/7f49e297-5386-415e-8121-79c2f8185553)
这谁打得过啊，比深渊都难打

**Comment by Kamisato100** [comment]
其他的都还能接受，那个新的帮派boss混合双打那关和黑化鼠妹各种掉人，感觉不应该把石头放在高难挑战里面。
本身你难不是问题，可以设置为无限复活，只要打过去就能拿齐菲林石头，大佬能无伤/限时/不死角色给一些额外狗粮丁尼技能奖励。
这样练度不够的或者手残党可以刮过去拿保底，喜欢高难挑战的可以选择凹无伤爽一把。

**Reply by misaka10384** [reply]
前面三关我确实凹爽了  直接崩三血脉觉醒  后面的BOSS太多了  无伤凹不动  后面两个与空洞为敌有三波  我死在第三波了

**Comment by ITwTIb** [comment]
蘸豆，爽！

**Comment by GWWA** [comment]
战斗！爽！

**Comment by Xuanan0** [comment]
什么空间站
<img width="72" alt="fb659f1e-0b41-4f90-9b21-274725fdb8d9" src="https://github.com/user-attachments/assets/59b77ba6-97ff-451f-a545-3817cfa4131a">
